src/usr/local/www/system_advanced_misc.php

```diff
@@ -541,6 +550,16 @@
 	'it can be restored automatically on the next boot. Keep in mind that the more '.
 	'frequent the backup, the more writes will happen to the media.');
 
+$section->addInput(new Form_Input(
+	'logsbackup',
+	'Periodic Logs Backup',
+	'number',
+	$config['system']['logsbackup'],
+	['min' => 0, 'max' => 24, 'placeholder' => 'Period between 1 and 24 hours']
+))->setHelp('This will periodically backup the logs so '.
+	'it can be restored automatically on the next boot. Keep in mind that the more '.
```

Comment:
Wording isn't quite right, either s/the logs/the log directory/ or s/it can/they can/